README.md

```diff
@@ -56,38 +72,38 @@ module "my-cluster" {
     }
   ]
 }
+
+resource "kubernetes_namespace" "example" {
+  metadata {
+    name = "example"
+  }
+}
 ```
+
 ## Conditional creation
 
 Sometimes you need to have a way to create EKS resources conditionally but Terraform does not allow to use `count` inside `module` block, so the solution is to specify argument `create_eks`.
 
-Using this feature _and_ having `manage_aws_auth=true` (the default) requires to set up the kubernetes provider in a way that allows the data sources to not exist.
-
+If you also want to sometimes create kubernetes resources then it gets slightly more complicated:
```

Comment:
I don't get your point.

Reply:
I was trying to provide an example of how to configure the data sources if the user wanted kubernetes resources to be optional. Maybe the wording needs clarifying? Or I could drop the section?

Reply:
I have tried rewording it